pkgs/development/python-modules/manticore/default.nix

```diff
@@ -0,0 +1,66 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, capstone
+, crytic-compile
+, ply
+, prettytable
+, psutil
+, pyelftools
+, pyevmasm
+, pysha3
+, pyyaml
+, rlp
+, setuptools
+, unicorn
+, wasm
+, wrapt
+, pkgs
+}:
+
+let z3 = pkgs.z3;
```

Comment:
Do you want ``pkgs.z3`` or ``pythonPackages.z3``? Would both/either work?

Reply:
I've tried `pythonPackages.z3` but couldn't get `z3` binary on the PATH.